modules/applications/pages/ui-builder/how-to-configure-https.adoc

```diff
@@ -0,0 +1,141 @@
+
+[source]
+----
+server_name bonitasoft.com;  # server_name directive: Sets the server domain.
+ssl_ciphers 'ECDHE-ECDSA-CHACHA20-POLY1305:ECDHE-RSA-CHACHA20-POLY1305:ECDHE-ECDSA-AES128-GCM-SHA256:ECDHE-RSA-AES128-GCM-SHA256:ECDHE-ECDSA-AES256-GCM-SHA384:ECDHE-RSA-AES256-GCM-SHA384:DHE-RSA-AES128-GCM-SHA256:DHE-RSA-AES256-GCM-SHA384:ECDHE-ECDSA-AES128-SHA256:ECDHE-RSA-AES128-SHA256:ECDHE-ECDSA-AES128-SHA:ECDHE-RSA-AES256-SHA384:ECDHE-RSA-AES128-SHA:ECDHE-ECDSA-AES256-SHA384:ECDHE-ECDSA-AES256-SHA:ECDHE-RSA-AES256-SHA:DHE-RSA-AES128-SHA256:DHE-RSA-AES128-SHA:DHE-RSA-AES256-SHA256:DHE-RSA-AES256-SHA:ECDHE-ECDSA-DES-CBC3-SHA:ECDHE-RSA-DES-CBC3-SHA:EDH-RSA-DES-CBC3-SHA:AES128-GCM-SHA256:AES256-GCM-SHA384:AES128-SHA256:AES256-SHA256:AES128-SHA:AES256-SHA:DES-CBC3-SHA:!DSS';
+ssl_prefer_server_ciphers on;
+ssl_session_cache shared:SSL:50m;
+ssl_session_timeout 5m;
+add_header Strict-Transport-Security max-age=15768000;
+ssl_certificate /etc/ssl/certs/nginx.crt;  # ssl_certificate directive: Specifies the path to the SSL certificate.
+ssl_certificate_key /etc/ssl/private/nginx.key;  # ssl_certificate_key directive: Specifies the path to the SSL certificate key.
+ssl_protocols TLSv1.2 TLSv1.3;  # ssl_protocols directive: Specifies the supported SSL protocols.
+ssl_dhparam /etc/ssl/certs/dhparam.pem;  # ssl_dhparam directive: Specifies the path to the Diffie-Hellman parameter file.
+----
+
+=== Secure the cookies
+
+The secure flag on Bonita Runtime `X-Bonita-API-Token` and `JSESSIONID` and Bonita UI Builder `SESSION` cookies should be set when using HTTPS.
+Activating the secure flag can be done by updating the nginx configuration of Bonita UI proxy.
+Update `nginx.conf` file and modify the `proxy_cookie_path` directive for the locations `/bonita/sessionLogout`, `/bonita/` and `/bonita/authservice` as such:
+[source]
+----
+proxy_cookie_path /bonita "/; Secure";
+----
+Also add the following line to the locations `~* ^/app/([^?/#]+)` and `\`:
+[source]
+----
+proxy_cookie_path / "/; secure";
+----
+
+=== Redirect HTTP to HTTPS
+
+To redirect HTTP requests to HTTPS, you need to update the nginx configuration of Bonita UI proxy.
+Update `nginx.conf` file to add the following server block:
+
+[source]
+----
+server {
+    listen 80;
+    server_name localhost;
+    return 302 https://$server_name$request_uri;
+}
+----
+
+== Configure Bonita Runtime
+
+=== Configure the environment variables
+
+In order to replace the apparent scheme in requests to Bonita Runtime (HTTP) with the one of the browser (HTTPS), you need to set the environment variable `REMOTE_IP_VALVE_ENABLED: "true"` in the Bonita Runtime service in the `docker-compose.yml` file:
+[source,yaml]
+----
+services:
+  ...
+  bonita:
+    image: ...
+    environment:
+      REMOTE_IP_VALVE_ENABLED: "true"
+----
+
+== Troubleshooting
```

Comment:
For Troubleshooting section, we got a pattern to follow to got the same display in each content.

```
[.troubleshooting-title]
== Troubleshooting

[.troubleshooting-section]
--
[.symptom]
I don't want display potential cause as title

[.symptom-description]
When i sure about the cause, I want to remove potential on title

[.cause]#Cause#
This is probably because I often go to bed at 3AM :'(

[.solution]#Possible solution#
Just write `Cause` as title
--
```


You can see more details and example in our [showroom](https://bonitasoft.github.io/bonita-documentation-theme/troubleshooting.html)